src/components/AddItemsToList.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
```

Comment:
I think in our PR we have used a capital C for our Components directory, I'm not sure what we had agreed on -  components, or Components? We can amend ours if we think lowercase makes more sense

Reply:
Thanks @rb50 . Since component names are capitalized, I think we leave directory names lowercase. My commit is a few minutes late. https://github.com/the-collab-lab/tcl-19-smart-shopping-list/pull/18/commits/b7817e3af92341534a25090c577ad6c7e639b9a6

Reply:
https://reactjs.org/docs/faq-structure.html In the official docs I see that the folder names are lowercase so I suggest we use that as a point of reference

Reply:
I would second Golfo and Joseph here.